pkg/apis/keda/v1alpha1/scaledobject.go

```diff
@@ -127,7 +127,7 @@ type ScaleTriggers struct {
 	// +optional
 	AuthenticationRef *ScaledObjectAuthRef `json:"authenticationRef,omitempty"`
 	// +optional
-	FallbackReplicas *int32 `json:"fallback,omitempty"`
```

Comment:
This is a breaking change in Keda API, usually you never remove fields without bumping the API version, instead you mark them as deprecated. Due to this, we need to update the Keda doc with the exact version which Flagger expects.